src/Sylius/Bundle/OrderBundle/Resources/config/services.xml

```diff
@@ -76,5 +76,10 @@
         <service id="Sylius\Component\Order\Remover\ExpiredCartsRemoverInterface" alias="sylius.expired_carts_remover" />
 
         <service id="sylius.factory.add_to_cart_command" class="Sylius\Bundle\OrderBundle\Factory\AddToCartCommandFactory" />
+
+        <service id="sylius.cart_changes_resetter" class="Sylius\Bundle\OrderBundle\ChangesResetter\CartChangesResetter">
```

Comment:
```suggestion
        <service id="sylius.resetter.cart_changes" class="Sylius\Bundle\OrderBundle\Resetter\CartChangesResetter">
```
Easier to upmerge since it'd be in line with the newer naming schema